lib/lints/avoid_late_keyword/models/avoid_late_keyword_parameters.dart

```diff
@@ -0,0 +1,20 @@
+/// A data model class that represents the "avoid late keyword" input
+/// parameters.
+class AvoidLateKeywordParameters {
+  /// Maximum number of parameters
+  final bool allowInitialized;
+
+  static const _defaultAllowInitialized = false;
+
+  /// Constructor for [AvoidLateKeywordParameters] model
+  const AvoidLateKeywordParameters({
+    required this.allowInitialized,
+  });
+
+  /// Method for creating from json data
+  factory AvoidLateKeywordParameters.fromJson(Map<String, Object?> json) =>
+      AvoidLateKeywordParameters(
+        allowInitialized: json['allow_initialized'] as bool? ??
+            _defaultAllowInitialized,
+      );
```

Comment:
Let's keep defaults in the constructor.
```suggestion
  /// Constructor for [AvoidLateKeywordParameters] model
  const AvoidLateKeywordParameters({
   this.allowInitialized = false,
  });

  /// Method for creating from json data
  factory AvoidLateKeywordParameters.fromJson(Map<String, Object?> json) =>
      AvoidLateKeywordParameters(
        allowInitialized: json['allow_initialized'] as bool?,
      );
```